install/installer/pkg/components/ide-service/ide_config_configmap.go

```diff
@@ -54,7 +54,7 @@ func ideConfigConfigmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 	jbPluginLatestImage := resolveLatestImage(ide.JetBrainsBackendPluginImage, "latest", ctx.VersionManifest.Components.Workspace.DesktopIdeImages.JetBrainsBackendPluginLatestImage)
 	jbLauncherImage := ctx.ImageName(ctx.Config.Repository, ide.JetBrainsLauncherImage, ctx.VersionManifest.Components.Workspace.DesktopIdeImages.JetBrainsLauncherImage.Version)
 	idecfg := ide_config.IDEConfig{
-		GpRunImage:      ctx.ImageName("registry.hub.docker.com", "gitpod/gp-run", "latest"),
+		GpRunImage:      ctx.ImageName("registry.hub.docker.com", "gitpod/gp-run", "ak-test"),
```

Comment:
Are we going to keep a fixed version here?
If we leave `latest` during the internal testing would allow us to publish live patches.

Reply:
It does not really matter, we can publish from ak/ide branch. It is reconfigured. We maybe will delete all of that.